test/functional/test_keras_framework.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import logging
+import os
+
+import numpy as np
+
+import sagemaker_containers as smc
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+RESOURCE_CONFIG = dict(current_host='algo-1', hosts=['algo-1'])
+
+INPUT_DATA_CONFIG = {'training': {'ContentType': 'trainingContentType',
+                                  'TrainingInputMode': 'File',
+                                  'S3DistributionType': 'FullyReplicated',
+                                  'RecordWrapperType': 'None'}}
+
+hyperparameters = dict(training_data_file='training_data.npz', sagemaker_region='us-west-2',
+                       default_user_module_name='net',
+                       sagemaker_job_name='sagemaker-training-job', sagemaker_enable_cloudwatch_metrics=True,
+                       sagemaker_container_log_level=logging.WARNING, sagemaker_program='user_script.py')
+
+USER_SCRIPT = """
+import os
+
+import keras
```

Comment:
why do we need to test it for a specific framework?
I don't think we have or should have any keras functionality in the container.

Reply:
SageMaker containers is independent of any specific framework. A framework is just needed for testing. These tests don't run in any container as well.